packages/cli/test/services/project-checker.test.ts

```diff
@@ -54,4 +61,171 @@ describe('project checker', (): void => {
             expect(exceptionThrown).to.be.equal(true)
         })
     })
+
+    describe('checkCurrentDirBoosterVersion', (): void => {
+        
+        beforeEach(() => {
+            replace(projectUpdater,'updatePackageJsonDependencyVersions', fake.resolves({}))
+            replace(logger,'info', fake.resolves({}))
```

Comment:
can you check that the linter isn't failing for this? 😮 
```suggestion
            replace(logger, 'info', fake.resolves({}))
```

Reply:
Remember to run `./scripts/check-all-the-things.sh` before submitting PRs

Reply:
I had to run `lerna run lint:check` to fix the linting errors. `./scripts/check-all-the-things.sh` wasn't enough